tests/test_mongo_doc_manager.py

```diff
@@ -35,15 +35,29 @@ def setUpClass(cls):
         MongoTestCase.setUpClass()
         cls.namespaces_inc = ["test.test_include1", "test.test_include2"]
         cls.namespaces_exc = ["test.test_exclude1", "test.test_exclude2"]
-        cls.choosy_docman = DocManager(
-            cls.standalone.uri,
-            namespace_set=TestMongoDocManager.namespaces_inc
-        )
 
     def setUp(self):
+       
+        """ setup instance variables for running doc manager test
+            using single meta collection or collection per namespace """
+        if type(self) is TestMongoDocManagerWithSingleMetaCollection:
```

Comment:
This seems a little messy. A suggestion: define `use_single_meta_collection` at the class level of `MongoTestCase` and set it to `False`. Below, define your class `TestMongoDocManagerWithSingleMetaCollection` as just setting `use_single_meta_collection` to `True`. When creating the DocManagers below, set `use_single_meta_collection` to `self.use_single_meta_collection`.